ui.go

```diff
@@ -63,9 +63,14 @@ func initNodesTable() *nodesView {
 		"MAX LATENCY"}
 
 	return &nodesView{t, header}
-
 }
-func initUI(backends []*Backend) {
+
+func initUI(lb *lb) {
```

Comment:
in the UI can we separately show two sites? @krishnasrinivas

Reply:
done, there is a formatting issue in the grid output, checking with @poornas